Sources/TransferMethod/SelectTransferMethodTypePresenter.swift

```diff
@@ -264,39 +262,36 @@ final class SelectTransferMethodTypePresenter {
         }
 
         for transferMethodType in result {
-            let transferMethodTypeDetail = transferMethodConfigurationKeyResult!
-                .populateTransferMethodTypeDetail(country: country,
-                                                  currency: currency,
-                                                  profileType: profileType,
-                                                  transferMethodType: transferMethodType)
+            let transferMethodTypeDetail = TransferMethodTypeDetail(currency, transferMethodType)
```

Comment:
No need of this since transferMethodTypes contains the details as well (processingTime, Fee and transferMethodType)

Reply:
fixed